src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -313,15 +328,14 @@ const TreeItem: React.FC<TreeItemProps> = props => {
         ]
       : []),
     {key: 'divider-4', type: 'divider'},
-    ...(remotePath
+    ...(remotePathUrl && remotePathUrl.includes('github')
```

Comment:
I think a better check here would be for `https://github.com` - there may be cases where for example in a gitlab repo the name to be `my-github-test` ...

Reply:
oh absolutely thanks mate 👍